diff --git a/pm_common/pmutil.c b/pm_common/pmutil.c
index a70fe2f..2a7769e 100755
--- a/pm_common/pmutil.c
+++ b/pm_common/pmutil.c
@@ -42,7 +42,7 @@ PMEXPORT PmQueue *Pm_QueueCreate(long num_msgs, int32_t bytes_per_msg)
     /* need extra word per message for non-zero encoding */
     queue->len = num_msgs * (int32s_per_msg + 1);
     queue->buffer = (int32_t *) pm_alloc(queue->len * sizeof(int32_t));
-    bzero(queue->buffer, queue->len * sizeof(int32_t));
+    memset(queue->buffer, 0, queue->len * sizeof(int32_t));
     if (!queue->buffer) {
         pm_free(queue);
         return NULL;
@@ -55,7 +55,7 @@ PMEXPORT PmQueue *Pm_QueueCreate(long num_msgs, int32_t bytes_per_msg)
             return NULL;
         }
     }
-    bzero(queue->buffer, queue->len * sizeof(int32_t));
+    memset(queue->buffer, 0, queue->len * sizeof(int32_t));
     queue->head = 0;
     queue->tail = 0;
     /* msg_size is in words */
